libs/sdk-backend-bear/src/backend/workspace/execution/preparedExecution.ts

```diff
@@ -57,6 +61,14 @@ export class BearPreparedExecution implements IPreparedExecution {
         return this.executionFactory.forDefinition(defWithDateFormat(this.definition, dateFormat));
     }
 
+    public withExecConfig(config: IExecutionConfig): IPreparedExecution {
+        invariant(
```

Comment:
Should this be a hard error? Or should we do a warning only and let the computation continue?

Reply:
It will warn the user and let computation continue as if they didn't set data sampling config

Reply:
That seems fair :)